credential/base/src/main/java/org/wildfly/security/credential/AbstractX509CertificateChainCredential.java

```diff
@@ -21,6 +21,7 @@
 import java.security.Provider;
 import java.security.cert.CertificateEncodingException;
 import java.security.cert.X509Certificate;
+import java.security.MessageDigest;
 import java.security.spec.AlgorithmParameterSpec;
 import java.util.Arrays;
```

Comment:
@mbhardwaj09 You have to remove this import of Arrays because it is not needed anymore

Reply:
@Skyllarr Removed it. Thank you!

Reply:
@Skyllarr, Removed it. Thank you!